src/routes/RoutesConfig.tsx

```diff
@@ -14,7 +16,9 @@ const RoutesConfig = () => {
     <Routes>
       <Route path="/" element={<Home />} />
       <Route path="blog" element={<Blog />} />
+      <Route path="blog/:blogId" element={<BlogArticle />} />
```

Comment:
Please use RoutePages enum for handling route paths. I've added example in the previous PR :)